en/02_Developer_Guides/05_Extending/00_Modules.md

```diff
@@ -34,12 +34,12 @@ enables you to install modules from specific versions, checking for compatibilit
 to track development branches of them. To install modules using this method, you will first need to setup Silverstripe CMS
 with [Composer](../../getting_started/composer).
 
-Each module has a unique identifier, consisting of a vendor prefix and name. For example, the "blog" module has the
-identifier `silverstripe/blog` as it is published by *Silverstripe*. To install, use the following command executed in
+Each module has a unique identifier, consisting of a vendor prefix and name. For example, the "linkfield" module has the
+identifier `silverstripe/linkfield` as it is published by *Silverstripe*. To install, use the following command executed in
 the project root folder:
 
 ```bash
-composer require silverstripe/blog
+composer require silverstripe/linkfield
 ```
```

Comment:
Unrelated but since blog isn't commercially supported I'm removing the reference.